src/Methods/Modular_Programming/Hard/Q1/RightPrimeTest.java

```diff
@@ -0,0 +1,45 @@
+package Methods.Modular_Programming.Hard.Q1;
+import global.BaseTest;
+import global.variables.*;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.RepetitionInfo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RightPrimeTest extends BaseTest{
+
+    public Clause[] testSentence() {
+        provideInput("5");
```

Comment:
I would just use the example number here, but it is up to you

Reply:
I try not to use example numbers in case of hard coding